documentation/using-azure-identity.md

```diff
@@ -0,0 +1,260 @@
+# Using @azure/identity with the Microsoft Identity Platform (AAD v2)
+
+This document intends to demystify the configuration and use of [Microsoft
+identity
+platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/),
+also known as Azure Active Directory v2, with the Azure SDK libraries.
+Microsoft identity platform implements the OAuth 2.0 and OpenID Connect
+standards to provide authentication for users and services who may be granted
+access to Azure services.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Understanding the Credential Types](#understanding-the-credential-types)
+- [Choosing a Credential Type](#choosing-a-credential-type)
+- [Credential Types in @azure/identity](#credential-types-in-azureidentity)
+
+## Getting Started
+
+Any application that must support authentication through Microsoft identity
+platform needs two things: a tenant and an app registration created for that
+tenant.
+
+A "tenant" is basically instance of Azure Active Directory associated with your
+Azure account.  You can follow the instructions on [this quickstart
+guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant)
```

Comment:
From an accessibility viewpoint, please consider

`You can follow the instructions on the [quick start guide for tenants](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-create-new-tenant) to either use an existing AAD tenant or create a new one.`

Reply:
Great point, I'll make this change and the other similar below